cli/remotebazel/remotebazel.go

```diff
@@ -912,14 +912,23 @@ func Run(ctx context.Context, opts RunOpts, repoConfig *RepoConfig) (int, error)
 	})
 	eg.Go(func() error {
 		var err error
-		exRsp, err = bbClient.GetExecution(ctx, &espb.GetExecutionRequest{ExecutionLookup: &espb.ExecutionLookup{
-			InvocationId: iid,
-		}})
-		if err != nil {
-			return fmt.Errorf("could not retrieve ci_runner execution: %s", err)
+		for i := 0; i < 5; i++ {
```

Comment:
can we use `WaitExecution` instead of polling?

Reply:
(`rexec.NewRetryingStream()` can help here)

Reply:
I started out on that approach, but the code feels a lot more complicated to introduce a stream. At this point, we know the execution is complete (it has stopped publishing event logs), so we won't have to poll for hours for example. What do you think of just going with the simpler polling approach?

Reply:
Ack - updated